qdrant_client/qdrant_remote.py

```
@@ -2209,14 +2210,56 @@ def _upload_collection(
+        batch_sizes = [size for size, _batch in sized_batches]
+        raw_batches = [batch for _size, batch in sized_batches]
+
         if parallel == 1:
             updater = self._updater_class.start(**updater_kwargs)
-            for _ in updater.process(batches_iterator):
-                pass
+            total_uploaded = 0
+            batch_count = 0
+            for size, result in zip(
+                batch_sizes, updater.process(raw_batches)
+            ):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)